packages/@o3r/workspace/schematics/library/index.spec.ts

```diff
@@ -17,6 +17,15 @@ jest.mock('@angular-devkit/schematics', () => {
 });
 
 const collectionPath = path.join(__dirname, '..', '..', 'collection.json');
+const angularJsonFile = `{
+  "version": 1,
+  "projects": {
+    "my-new-module": {
+      "projectType": "library",
+      "root": "packages-test/my-new-module"
+    }
+  }
+}`;
```

Comment:
```suggestion
const angularJsonFile = JSON.stringify({
  version: 1,
  projects: {
    'my-new-module': {
      projectType: 'library',
      root: 'packages-test/my-new-module'
    }
  }
});
```
would be more colorful :)

Reply:
this could even be helpful with actual `angular.json` typing (if such thing exists)